### PR TITLE
Added quickobs

### PR DESCRIPTION
**Pull request by philclifford**
since I never got to sort my Mullvad VPN fix update for you (thanks for picking it up) at least I can try and save you having to deal with this one from Yannick :smile:

### CODE REVIEW

File: 01-main/packages/quickobs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+CODENAMES_SUPPORTED="focal jammy kinetic"
+ARCHS_SUPPORTED="amd64"
```
Comment (natanjunges) [inline 01-main/packages/quickobs:3]: This is already the default, so you can remove it.

**Review by natanjunges — CHANGES_REQUESTED** [review]

**Comment by natanjunges** [comment]
The test is failing because it is trying to install other packages, perhaps because you need to resynchronize your branch. We should try to improve the test so that it does not happen.

**Review by natanjunges — APPROVED** [review]

**Comment by natanjunges** [comment]
https://github.com/natanjunges/deb-get/actions/runs/3269581986/jobs/5377243956#step:5:20
`wavebox` *is* being detected as modified (and so are `lutris`, `vivaldi-stable` and `webex`), and it is not a dependency of `quickobs`.

**Comment by philclifford** [comment]
hmm .... PR showed only changes in quickobs ... that branch had synced your commits re webex, vivaldi etc ... but the test decided they were "changed" .  Rebased and pushed : same (but test passed) .  Rebased again (using gitkraken)  (lots of skipping of "already applied patch") - pushed.  Test ran again and only detected quickobs this time (yay!)  
Odd.  

image

Anyway,  sorted now I hope - sorry for the hassle and my confusion.

**Comment by natanjunges** [comment]
I think there might be a better git magic that would not have this problem, but I need more experimenting.

**Pull request merged**